src/stack_mitosis/interpreter.clj

```diff
@@ -78,8 +77,8 @@
 (comment
```

Comment:
Can you remind me how these comments work?

If I remember correctly, these are used for repl driven development locally, but don't have any effect during the actual run?

Reply:
yep, comment blocks are ignored at normal evaluation, but can be run manually by the developer. So generally the concerns on comment block code are if it's escaping secrets or the like.